code/game/gamemodes/changeling/powers/glands.dm

```diff
@@ -9,5 +9,5 @@
 	..()
 	var/datum/changeling/changeling=user.mind.changeling
 	changeling.chem_storage += 25
-	changeling.chem_recharge_rate *=2
-	return
\ No newline at end of file
+	changeling.chem_recharge_rate *=1.5

```

Comment:
https://github.com/tgstation/-tg-station/pull/16457#issuecomment-202592392
should be in the desc but they were mentioned